modules/justpremiumBidAdapter.js

```diff
@@ -54,6 +62,13 @@ export const spec = {
 
     const payloadString = JSON.stringify(payload)
 
+    LAST_PAYLOAD = payload
+    $$PREBID_GLOBAL$$.onEvent('bidWon', (data) => {
```

Comment:
@nwlosinski `$$PREBID_GLOBAL$$` usage is not allowed in adapter except for prebid verion.  You can drop pixels using http://prebid.org/dev-docs/publisher-api-reference.html#setConfig-Configure-User-Syncing

Reply:
@jaiminpanchal27 thanks for the link but we didn't add this to sync users. We need to gather some additional information to be able better detects why we lose. If we are not allowed to use `$$PREBID_GLOBAL$$` how we can achieve that? Is there any other way to send some tracking when action ends?

Reply:
@jaiminpanchal27 are you able to provide any input here?

Reply:
@nwlosinski Sorry this didn't came to my mind earlier but you can add `onTimeout` and `onBidwon` function to your bidder spec and achieve this. Docs are here http://prebid.org/dev-docs/bidder-adaptor.html#registering-on-timeout

Some of the bidders are using these functions you can check those for reference.

Reply:
@jaiminpanchal27 yeah but `onBidWon` listener is executed only when we won the bidding where we wanted to send tracking pixel if other parties did that.
So I understand that there is no way that you can accept this PR in current shape?

Reply:
@mkalafior yes we cannot as of now.

Reply:
@jaiminpanchal27 thanks for your time. For now we removed this part.